src/curve/stable/StablePair.sol

```diff
@@ -109,7 +109,11 @@ contract StablePair is ReservoirPair {
     function mint(address) external override returns (uint256) {
         // DELEGATE TO StableMintBurn
         address lTarget = MINT_BURN_LOGIC;
-        assembly {
+
+        // SAFETY:
+        // This assembly block is memory safe as it does not manipulate the free memory pointer
+        // and that the calldata and return value each do not exceed 64 bytes
```

Comment:
Why does calldata/returndata not exceed 64 bytes, it could easily exceed 64 bytes right?

This block is memory safe because it does not return back to solidity (see the in asm revert and return)

Reply:
I see. So whenever we return/revert from the assembly with a wonky free memory pointer, it is still memory safe? Cuz after the return, the caller function would have its own memory space / context and that's not been affected? 

For discussion purposes, in this case of `mint/burn` `delegatecall`, both functions' args are just address (within 32 bytes) and the return data are either 32 bytes for mint or 64 bytes for burn. I notice that both don't exceed the 64 bytes of "scratch space" (from 0x00 - 0x3f), that's why I made the comment